src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderList.jsx

```diff
@@ -0,0 +1,275 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
+import recordEvent from 'platform/monitoring/record-event';
+import {
+  FETCH_STATUS,
+  FACILITY_SORT_METHODS,
+  GA_PREFIX,
+} from '../../../utils/constants';
+import { updateCCProviderSortMethod } from '../../redux/actions';
+import ResidentialAddress from '../../../components/ResidentialAddress';
+import InfoAlert from '../../../components/InfoAlert';
+import LoadProvidersErrorAlert from './LoadProvidersErrorAlert';
+import NoProvidersAlert from './NoProvidersAlert';
+
+export default function ProviderList({
```

Comment:
Looking at these props, it seems like I could use the same selector method as `ProviderSelectionField` to select the state needed instead of passing them directly to the component.

Reply:
Updated both `ProviderList` and `SelectedProvider` to use the `useSelector` hook and reduce the number of props I'm passing. I used the same selector (`selectProviderSelectionInfo`) for all three components in this PR and picked out the state that was needed. Wondering if that's a bit of an anti-pattern in Redux.